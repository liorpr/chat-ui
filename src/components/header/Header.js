
import React from 'react';
import { Container, Image } from 'semantic-ui-react';
import logo from '../../assets/spotim-logo.jpg';

const Header = () => (
  <Container className="spotim-header">
    <div className="spotim-title">
      Welcome to the Spot.IM Chat app
    </div>
    <div>
      <div>
        <Image size={'tiny'} src={logo}/>
      </div>
    </div>
  </Container>  
);

export default Header;