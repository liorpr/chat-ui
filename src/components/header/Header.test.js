
import React from 'react';
import TestRenderer from 'react-test-renderer';
import Header from './Header';

it('renders correctly', () => {
  const tree = TestRenderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});