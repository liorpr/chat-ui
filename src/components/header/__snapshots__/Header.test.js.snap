// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui container spotim-header"
>
  <div
    className="spotim-title"
  >
    Welcome to the Spot.IM Chat app
  </div>
  <div>
    <div>
      <img
        alt={undefined}
        className="ui tiny image"
        height={undefined}
        src="spotim-logo.jpg"
        width={undefined}
      />
    </div>
  </div>
</div>
`;
